backends/qualcomm/tests/test_qnn_delegate.py

```diff
@@ -5758,7 +5758,13 @@ def setup_environment():
         default="",
         type=str,
     )
-
+    parser.add_argument(
+        "-t",
+        "--target",
+        help="Target platform for deployment",
+        default="aarch64-android",
```

Comment:
Can we add choices so users know what they can choose? 
```suggestion
        choices=["aarch64-android", "linux"]
        default="aarch64-android",
```

Reply:
That's even better, thank you.